iree/compiler/Dialect/VM/Conversion/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+iree_cc_library(
+  NAME
+    ImportUtils
+  HDRS
+    "ImportUtils.h"
+  SRCS
+    "ImportUtils.cpp"
+  DEPS
+    iree::compiler::Dialect
+    iree::compiler::Dialect::VM:IR
+    MLIRIR
+    MLIRParser
+    MLIRSupport
+    MLIRTransforms
+  ALWAYSLINK
+  PUBLIC
+)
```

Comment:
can you add a newline here at the end of the file?
(I had thought our clang-format presubmit would catch this but I guess it makes sense it doesn't run on .txt files -- may want to set your editor to ensure newlines until we can fix that :)

Reply:
Yeah, I missed that one. Fixed.